core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -652,24 +640,24 @@ func (c *coordinator) unmarshalLogs(
 				return
 			}
 			randomWordsFulfilledLogs = append(randomWordsFulfilledLogs, rwf)
-		case c.newTransmissionTopic:
+		case c.outputsServedTopic:
 			unpacked, err2 := c.onchainRouter.ParseLog(rawLog)
 			if err2 != nil {
 				// should never happen
-				err = errors.Wrap(err2, "unmarshal NewTransmission log")
+				err = errors.Wrap(err2, "unmarshal OutputsServed log")
 				return
 			}
-			nt, ok := unpacked.(*vrf_beacon.VRFBeaconNewTransmission)
+			nt, ok := unpacked.(*vrf_coordinator.VRFCoordinatorOutputsServed)
 			if !ok {
 				// should never happen
-				err = errors.New("cast to *vrf_beacon.VRFBeaconNewTransmission")
+				err = errors.New("cast to *vrf_coordinator.VRFCoordinatorOutputsServed")
 			}
-			newTransmissionLogs = append(newTransmissionLogs, nt)
+			outputsServedLogs = append(outputsServedLogs, nt)
 		default:
 			c.lggr.Error(fmt.Sprintf("Unexpected event sig: %s", lg.EventSig))
 			c.lggr.Error(fmt.Sprintf("expected one of: %s %s %s %s",
 				hexutil.Encode(c.randomnessRequestedTopic[:]), hexutil.Encode(c.randomnessFulfillmentRequestedTopic[:]),
-				hexutil.Encode(c.randomWordsFulfilledTopic[:]), hexutil.Encode(c.newTransmissionTopic[:])))
+				hexutil.Encode(c.randomWordsFulfilledTopic[:]), hexutil.Encode(c.outputsServedTopic[:])))
```

Comment:
Minor: can we combine this into one log and add log names as well as the topics?

```golang
c.lggr.Error(fmt.Sprintf("expected one of: %s (RandomnessRequested) %s (RandomnessFulfillmentRequested) %s (RandomWordsFulfilled) %s (OutputsServed), got %s", 
  hexutil.Encode(c.randomnessRequestedTopic[:]), 
  hexutil.Encode(c.randomnessFulfillmentRequestedTopic[:]),
  hexutil.Encode(c.randomWordsFulfilledTopic[:]), 
  hexutil.Encode(c.outputsServedTopic[:]), 
  lg.EventSig))
```